src/po/Makefile

```diff
@@ -327,7 +328,6 @@ $(PLUGPACKAGE).pot: $(PO_PLUG_INPUTLIST)
 	rm -f *.js ./vim_to_js
 
 # Converting the PO file of the plug-in package to the binary format of the MO
-MO_PLUGPACKAGE_PATH != pwd
 $(PLUGPACKAGE).mo: $(PO_PLUGPACKAGE)
 	$(MSGFMTCMD) -o $(MO_PLUGPACKAGE_PATH)/$@ $<
```

Comment:
Should be `POT_PLUGPACKAGE_PATH` instead of `MO_PLUGPACKAGE_PATH`?

Reply:
Yes, I hurried, inattentive.